app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,12 @@
+class FeedbacksController < ApplicationController
```

Comment:
Warum plural? Ist das nicht eher ein FeedbackController

Reply:
Das hat das generate so benannt, sollte also passen.

Reply:
Look at this https://guides.rubyonrails.org/getting_started.html#adding-a-second-model